vvydier/iOS
# Try OpenTelemetry API/SDK to add instrumentation to the DDG app

Task/Issue URL: Add Opentelemetry-swift instrumentation to the DDG swift app
Tech Design URL: Added opentelelemtry instrumentation and collect traces/spans
CC:

**Description**:
Add opentelemetry-swift dependencies and add instrumentation for network calls in the DDG iOS app

**Steps to test this PR**:
1. Run the iOS app, test it with a few network calls. Check if the traces and spans are collected and displayed with a Console Exporter or Jaegar/Zipkin



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:
